### PR TITLE
[CCP] Refactored Treatment Outcomes Table and Icons to Remove Included/Compared Treatments

### PR DESCRIPTION
**Pull request by noranda**
This PR satisfies JIRA task [2036](https://jira.mitre.org/browse/ASCODCP-2036).

- Removed all references to "included" or "compared" treatments, generalizing to "similar patient" treatments
- Left some commented out code that will help with future tasks for adding ability to compare treatments
- Removed included treatment table
- Removed selection of treatments (will be added back using different logic in a future task)
- All cohort treatment combinations are now displayed by default
- Icons view was refactored slightly to fix after included treatment was removed -- a future task will implement this view further to be in line with [design](
https://www.figma.com/proto/7y2xB3nMuEuWIwWlgjejZi/CCP-cancerlinq-limited?node-id=555%3A6877&viewport=1190%2C741%2C0.5&scaling=min-zoom)
- Updated styles for new table to be more in line with design
- Updated tests

_Pull requests into Flux require the following. Submitter should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable (**N/A**) and :white_check_mark:. Reviewers should complete every item on the bulleted list when reviewing._

## Submitter:

**Running the Application**

- [x] Manually tested in Chrome (primary browser for testing)
- [x] Manually tested in IE/Safari (verify app loads and basic feature testing)
- [ ] Manually tested in Chrome in iPad mode (at minimum, viewing patient record)

**Task**

- [x] Branch implements task as expected
- [ ] Key application features are updated as needed for task and tested (see: [Key Application Features](https://github.com/FluxNotes/flux/wiki/Key-Application-Features))
- [ ] Task specific definition of done (as described in the Jira task) is met, if applicable.

**Documentation**

- [x] This pull request describes why these changes were made
- [x] Recognizes any potential shortcomings/bugs in the description 
- [ ] Pre release script is updated on the [Pre Release Testing Script wiki page](https://github.com/FluxNotes/flux/wiki/Pre-Release-Testing-Script) (add/remove single features to test, do not run through full script)
- [ ] Update the list of key features with new features on the [Key Application Features wiki page](https://github.com/FluxNotes/flux/wiki/Key-Application-Features)
- [ ] Document any new APIs/extension points
- [ ] Additional technical components and libraries are documented and added to [wiki](https://github.com/FluxNotes/flux/wiki/About-Flux-Notes) as needed

**Code Quality**

- [x] Code style guide is followed (see: [Code Style Guide](https://github.com/FluxNotes/flux/wiki/JavaScript-and-HTML-Code-Style-Guide))
- [x] Code is commented

**Tests**

- [x] Existing tests passed
- [ ] Added backend tests for new functionality added

## Reviewers:

- Check that code is maintainable and reusable. Code reuses existing code and infrastructure where appropriate.
- Check the PR and code accomplishes the task's purpose.
- Check that new tests appropriately test the new code, including edge cases. Other existing tests pass.
- Verify Pull Request checklist is correctly completed by submitter.
- Try to break the code

### CODE REVIEW

File: src/mcode-pilot/components/TreatmentOptionsOutcomesIcons/TreatmentOptionsOutcomesIcons.js [hunk 1]
```diff
@@ -11,142 +11,53 @@ export default class TreatmentOptionsOutcomesIcons extends Component {
...
-                normalizedSideEffects.push({
-                    name: key,
-                    numEffects: Math.round(effects[key] / totalPatients * 100)
-                });
-            });
-            normalizedSideEffects.sort((a, b) => b.numEffects - a.numEffects);
-        }
+        const timescale = survivalMap[timescaleToggle];
+        const similarPatientTreatmentNames = _.map(similarPatientTreatmentsData, 'displayName');
+        const numSurvive = Math.floor(selectedTreatment[timescale] / selectedTreatment.totalPatients * 100);
```
Comment (KeeyanGhoreshi) [inline src/mcode-pilot/components/TreatmentOptionsOutcomesIcons/TreatmentOptionsOutcomesIcons.js]: [KG1] If no treatment is "selected", or in this case if you select enough filters such that no patients meet the criteria and try switch to icon view, `selectedTreatment` is undefined and the error crashes the app.

**Comment by dreeves4321** [comment]
(not sure if it's part of this PR, but) [DR1] the table key indicates that white is "cancer related deaths" which is probably not accurate. Should just be "all deaths" (the typo was a holdover from when we thought we could distinguish).

**Comment by noranda** [comment]
[KG1] and [DR1] fixed. Branch is up to date with `master`. Thanks!

**Comment by KeeyanGhoreshi** [comment]
[KG2] The icons view doesn't update when you change filters.  You have to switch treatments to see it update.  It works on [https://fluxnotes.org/ccp](https://fluxnotes.org/ccp).

**Comment by noranda** [comment]
[KG2] Fixed. Also moved sorting logic to `TreatmentOptionsOutcomes` component. Sorting of icons view will now match table view, so when switching between them the treatments will line up. Should align a little better once we implement the rest of the changes for the icons view.

**Pull request merged**